components/navigation-bar.css

```diff
@@ -0,0 +1,73 @@
+.nav-container {
+    display: flex;
+    padding: 0.25rem;
+    box-shadow: 0 0 10px 4px rgb(0 0 0 / 5%);
+}
+
+.nav-logo {
+    font-family: 'Roboto Slab', serif;
+    padding: 0.5rem 1rem;
+    
+}
+.nav-logo a {
+    text-decoration: none;
+}
+.nav-title {
+    font-size: 2rem;
+    font-weight: 500;
+    color: var(--color-darker-gray);;
+
+}
+
+.nav-U-I {
+    font-weight: 600;
+}
+
+
+.nav-tagline {
+    font-weight: 500;
+    font-size: 1rem;
+    color: #1f1f1f;
+}
+
+.nav-links {
+    
+}
```

Comment:
It is not a good practice to define empty classes. You can either comment it out or remove it.